src/sqlparser/src/parser.rs

```diff
@@ -3649,6 +3649,40 @@ impl Parser<'_> {
                 (Keyword::SYSTEM_TIME, Keyword::AS, Keyword::OF),
                 cut_err(
                     alt((
+                        preceded(
+                            (
+                                Self::parse_identifier
+                                    .verify(|ident| ident.real_value() == "current"),
```

Comment:
Let's use `now()` instead of `current()`, since `now()` could be used in the batch query directly and process time temporal join is no longer using `now` as key word.